x-pack/plugins/apm/public/components/app/ServiceDetails/ServiceIntegrations/MachineLearningFlyout.tsx

```diff
@@ -95,7 +94,7 @@ export class MachineLearningFlyout extends Component<FlyoutProps, FlyoutState> {
 
   public addErrorToast = () => {
     const { urlParams } = this.props;
-    const { serviceName = 'unknown' } = urlParams;
+    const { serviceName } = urlParams;
```

Comment:
@jasonrhodes Just stumbled upon this. I don't think we should assign default values to `urlParams` like this. Do you remember why it was done this way?

Reply:
Not sure, probably a left-over from the metrics stuff where I didn't totally understand how urlParams needed to work